workshop/content/40-dotnet/70-advanced-topics/100-pipelines/3000-new-pipeline.md

```diff
@@ -59,12 +59,11 @@ namespace CdkWorkshop
                     SourceArtifact = sourceArtifact,  // Where to get source code to build
                     CloudAssemblyArtifact = cloudAssemblyArtifact,  // Where to place built source
 
-                    InstallCommands = new [] 
-                    {
-                        "npm install -g aws-cdk"
-                    },
-                    BuildCommands = new [] { "dotnet build src" }, // Language-specific build cmd
-                    SynthCommand = "cdk synth"
+                    InstallCommand = string.Join(
```

Comment:
See my comment on #175 about only needing `"npm install -g aws-cdk"`.

Also, that [String.Join Method](https://docs.microsoft.com/en-us/dotnet/api/system.string.join?view=netcore-3.1) signature looks off.  I was looking for the ` && ` separator somewhere like in this code:

```csharp
InstallCommand = string.Join(" && ",
	new string [] {
		"npm install -g aws-cdk",
		"sudo apt-get install -y dotnet-sdk-3.1"
	}
```